rust/agents/relayer/src/msg/metadata/base.rs

```diff
@@ -131,7 +131,7 @@ impl AppContextClassifier {
     pub async fn get_app_context(
         &self,
         message: &HyperlaneMessage,
-        root_ism: H256,
+        root_ism: Option<H256>,
```

Comment:
making this optional, because this is irrelevant to submitter queue messages. Maybe we'll switch to a fancier filter type that can take in `ism`s as well as other things if this feature gets more use